core/src/main/java/org/apache/iceberg/rest/requests/UpdateTableRequest.java

```diff
@@ -84,6 +98,7 @@ public static class Builder {
     private final List<MetadataUpdate> updates = Lists.newArrayList();
     private final Set<String> changedRefs = Sets.newHashSet();
     private final boolean isReplace;
+    private TableIdentifier identifier;
```

Comment:
I prefer to specifically default to null so it is clear when you read the class.

Reply:
done